### PR TITLE
 Links: .NET (2021-03)

### PR DESCRIPTION
**Pull request by DCtheGeek**
Monthly links cleanup.

### CODE REVIEW

File: docs/csharp/misc/cs1668.md [hunk 1]
```diff
@@ -16,5 +16,5 @@ Invalid search path 'path' specified in 'path string' --  'system error message'
   
 ## See also
 
-- [C# Compiler Options Listed Alphabetically](../language-reference/compiler-options/listed-alphabetically.md)
+- [C# Compiler Options Listed Alphabetically](../language-reference/compiler-options/index.md)
```
Comment (Youssef1313) [inline docs/csharp/misc/cs1668.md:19]: Link text is now outdated.

File: docs/csharp/programming-guide/main-and-command-args/top-level-statements.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ You can write a `Main` method explicitly, but it can't function as an entry poin
 
 > CS7022 The entry point of the program is global code; ignoring 'Main()' entry point.
 
-In a project with top-level statements, you can't use the [-main](../../language-reference/compiler-options/main-compiler-option.md) compiler option to select the entry point, even if the project has one or more `Main` methods.
+In a project with top-level statements, you can't use the [-main](../../language-reference/compiler-options/advanced.md) compiler option to select the entry point, even if the project has one or more `Main` methods.
```
Comment (Youssef1313) [inline docs/csharp/programming-guide/main-and-command-args/top-level-statements.md:35]:
```suggestion
In a project with top-level statements, you can't use the [-main](../../language-reference/compiler-options/advanced.md#mainentrypoint-or-startupobject) compiler option to select the entry point, even if the project has one or more `Main` methods.
```

File: docs/index.yml [hunk 3]
```diff
@@ -278,7 +281,7 @@ additionalContent:
             text: Get started in 5 minutes
           - url: iot/tutorials/blink-led.md
             text: Blink an LED
-          - url: https://aka.ms/IoTNet101
+          - url: <https://aka.ms/IoTNet101>
```
Comment (Youssef1313) [inline docs/index.yml:284]: May need confirmation on staging that usage of markdown link syntax still works in YAML.

Comment (IEvangelist) [reply docs/index.yml:284]: This doesn't work, and correctly triggers a build warning:
image

Comment (IEvangelist) [inline docs/index.yml:284]:
```suggestion
          - url: https://aka.ms/IoTNet101
```

File: docs/csharp/misc/cs1668.md (hunk 1 above)
Comment (IEvangelist) [inline docs/csharp/misc/cs1668.md:19]:
```suggestion
- [C# Compiler Options](../language-reference/compiler-options/index.md)
```

**Review by IEvangelist — APPROVED** [review]

File: docs/index.yml (hunk 3 above)
Comment (DCtheGeek) [reply docs/index.yml]: It's markdownlint that does that and I just missed cleaning it up after running it to do cleanup. ;)

**Pull request merged**